src/main/webapp/resources/js/pages/analysis/components/AnalysisSteps.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { Steps } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+const Step = Steps.Step;
+
+export function AnalysisSteps() {
+  const { analysisContext } = useContext(AnalysisContext);
+  return (
+    <>
+      <Steps
+        current={analysisContext.stateMap[analysisContext.analysisState]}
+        status={analysisContext.isError ? "error" : "finish"}
+        style={{ paddingBottom: "15px" }}
```

Comment:
Standard spacing

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a